wp-content/themes/missilethreat/template-parts/block-post-featured.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * The default template for displaying content
+ *
+ * Used for both singular and index.
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package CSIS iLab
+ * @subpackage @package MissileThreat
+ * @since 1.0.0
+ */
+
+?>
+
+<article <?php post_class('featured-post'); ?> id="post-<?php the_ID(); ?>">
+
+	<?php
+		the_post_thumbnail(array(400, 304));
+	?>
+
+	<div>
+
+		<?php
+			foreach (get_the_category() as $category) {
```

Comment:
I would move this to the `template-tags.php` file and make it a reusable function, that way if you have to edit the markup, you only have to do it in one place.